tests/test_citeprocpy.py

```diff
@@ -29,17 +40,19 @@ def test_failing_citations(self):
 
     def test_passing_citations(self):
         node = Node()
+        node.visible_contributors = OSFUser.objects.filter(fullname='Henrique Harman')
         url_data_path = os.path.join(os.path.dirname(__file__), '../website/static/citeprocpy_test_data.json')
         with open(url_data_path) as url_test_data:
             data = json.load(url_test_data)['passes']
         not_matches = []
+        citation = []
         for k, v in data.iteritems():
             try:
                 citeprocpy = render_citation(node, k)
```

Comment:
It doesn't look like the `mla` and `chicago` citation types are being tested here since `render_citation` makes an `apa` citation by default.

Reply:
Discussed in person. All citation formats are being tested.